src/components/AddForm.vue

```diff
@@ -285,6 +297,13 @@ export default class AddForm extends Vue {
     this.$refs.fileZone.addEventListener('drop', this.onDrop, true);
   }
 
+  @Watch('state', {immediate: true})
```

Comment:
```ts
@Watch('state', {deep: true})
onStateUpdate(state) {
  if (state.downloadItem) {
    this.setParams('urls', state.downloadItem.url);
  }
}
```

Reply:
Thank you! I am quite new to Vue, learning on the fly.